db/migrate/20220527183807_create_ltis.rb

```diff
@@ -0,0 +1,22 @@
+class CreateLtis < ActiveRecord::Migration[7.0]
+  def change
+    create_table :lti_clients do |t|
+      t.string :client_id
```

Comment:
@pretendWhale my only comment is that it seems all of these columns (across the three tables) should be non-null? Certainly the associations should all exist, otherwise the data is in an inconsistent state?

Reply:
Great point, and agreed. I've added those constraints